### ISSUE TITLE
Issue with parameter input for the Groups and Integrations Calculator. 

### ISSUE THREAD

**Issue by julesfowler** [ExoCTK/exoctk#281]
When testing the new release on `tlexo` -- going to the groups and integrations calculator results in an "Internal Service Error". 

I haven't added any new science content or features to the tool, so I think this must be a problem with how either the form validation or the API stuff was incorporated. 

The error seems to be that it is requesting arguments and trying to execute them before the form even loads -- which seems odd. Please @mfixstsci or @hover2pi HALP -- I'm not really sure what's changed about the form validation / API stuff. 

The error we see on the backend is : 
```Traceback (most recent call last):
  File "/internal/data1/anaconda3/envs/exoctk-3.6/lib/python3.6/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/internal/data1/anaconda3/envs/exoctk-3.6/lib/python3.6/site-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/internal/data1/anaconda3/envs/exoctk-3.6/lib/python3.6/site-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/internal/data1/anaconda3/envs/exoctk-3.6/lib/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/internal/data1/anaconda3/envs/exoctk-3.6/lib/python3.6/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/internal/data1/anaconda3/envs/exoctk-3.6/lib/python3.6/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/internal/data1/exodeploy/exoctk/exoctk/exoctk_app/app_exoctk.py", line 275, in groups_integrations
    trans_dur = float(request.args.get('transit_duration'))
TypeError: float() argument must be a string or a number, not 'NoneType'
```

**Comment by julesfowler** [comment]
@mfixstsci @hover2pi 

Yep -- playing with this a little more. The error is coming because the form is trying to execute before there's anything in it, and then the math and type conversion is useless on a `NoneType`. 

How can we stop this? Do I need default parameters? Is there a way to tell it not to execute until we give it input?

**Comment by mfixstsci** [comment]
I am working on this right now, do you see what the value is for `transit_duration` in the url params for the plant you are looking at? I am writing a little else if block to handle these situations.

**Comment by julesfowler** [comment]
I'm not looking at a planet because the form breaks before you can enter any info into the tool.
Like, you click on "Groups and Integrations Calculator" and it goes to internal service error.

**Comment by mfixstsci** [comment]
Okay @julesfowler, I will be submitting a PR again shortly.

**Comment by julesfowler** [comment]
@mfixstsci Here's a tidbit. The snippet is trying to request `transit_duration` from the form, but that isn't in the `groups_integrations.html`, nor is it in the `target_resolve.html`. Does this need to be a parameter that you fill out in the `html` form, or should this be populated by an ExoMAST query?

**Comment by julesfowler** [comment]
This is closed with #283
